templates/server.properties.j2

```diff
@@ -5,8 +5,10 @@
 
 # The id of the broker. This must be set to a unique integer for each broker.
 # If left off one is auto-assigned
-{% if kafka.broker_id is defined %}
-broker.id={{ kafka.broker_id }}
+{% if kafka.aws_cluster_autodiscover.data is defined %}
```

Comment:
Can this be written 
``broker.id={{ kafka.aws_cluster_autodiscover.data.id | default(kafka.id) }}``